tests/alive-tv/memory/gep-struct1.src.ll

```diff
@@ -0,0 +1,9 @@
+%0 = type { i32, i128, i16 }
```

Comment:
these tests need a data layout string otherwise they may on machines with different CPUs and/or ABIs

Reply:
@pranavk Adding `target datalayout = "e-i32:32-i128:32-i16:16"` at the top of gep-struct1.src.ll (and gep-struct2.src.ll as well) will work.

Reply:
To add a negative test, copying the src/tgt files to `gep-struct1-fail.<src/tgt>.ll`, changing `20` in tgt with a bogus number, and adding
```
; ERROR: Value mismatch
```
at the end of `gep-struct1-fail.src.ll` will work.

Reply:
yes, will work on this PR later today or tomorrow.